### DISCUSSION TITLE
cfclient print errors

### DISCUSSION THREAD

**Discussion by wiedenmayerjulian** [bitcraze/discussions#1697] [General]
Hello together!

When printing is too much for cfclient, these errors appear:

<img width="381" alt="image" src="https://github.com/user-attachments/assets/910d2319-d517-4da0-a4ac-98a437bb3ada" />

("\<F\>" and a "square character" i cannot copy)

What can i do to prevent this? Sometimes i need a high amount of prints.

Is there any configuration I can do to maximize the possible print without an error?

Thank you very much in advance

**Comment by ToveRumar** [comment]
Hi @jwdev297!

This probably happens because data is dropped when the traffic gets to jammed. It is hard to do anything about this except print at a lower rate. 

One idea could be to use our logging framework instead
It looks like this type of print could be good to visualize in another format, such as a plot. 
Our logging framework is built to handle higher rates of data so it will probably work better, even though I do not know exactly at what speed you need the data to be logged. 
This is how you add a variable to the log framwork https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/userguides/logparam/ 

In the client you can then look at the logs like this; 
https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/logblocks_tab/

I hope this helps you in your use case!
